integration-test/tests/cms/codeofconduct.test.ts

```diff
@@ -0,0 +1,17 @@
+import {expect, test} from '@playwright/test';
+import { codeofconductExepctedInformation } from '../../utils/datafactory/codeofconduct.data';
+test('GET /api/cms/v1/code-of-conduct returns correct team data', async ({request}) => {
+    const response = await request.get(`https://wcc-backend.fly.dev/api/cms/v1/code-of-conduct`);
```

Comment:
Please check the example from @nora-weisser which you can only use the base path.

Reply:
@dricazenck - When I use the Base URL for my .env file API_HOST="https://wcc-backend.fly.dev" then test are failing 
![image](https://github.com/user-attachments/assets/bae259e1-0dde-407e-884b-7f06c6be2cfe) not sure if I need to do any changes in any other file?